specification/purview/data-plane/Azure.Analytics.Purview.SelfServicePolicies/preview/2022-12-01-preview/purviewSelfServicePolicy.json

```diff
@@ -0,0 +1,348 @@
+          "minLength": 1,
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message",
+          "minLength": 1,
+          "type": "string"
+        },
+        "details": {
+          "description": "The error details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorModel"
+          }
+        }
+      }
+    },
+    "ErrorResponseModel": {
+      "description": "The error response model for policy",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/ErrorModel"
+        }
+      }
+    },
+    "SelfServicePolicy": {
+      "description": "The Self-service policy model",
+      "required": [
+        "decisionRule",
+        "workflowRunId"
+      ],
+      "type": "object",
+      "properties": {
+        "workflowRunId": {
+          "type": "string",
+          "description": "The workflowRunId of the workflow which created the policy"
+        },
+        "requestor": {
+          "type": "string",
+          "description": "The requestor of the self-service policy"
+        },
+        "expiryDate": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Expiry Date of the policy"
+        },
+        "decisionRule": {
+          "$ref": "#/definitions/SelfServicePolicyDecisionRule"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      }
+    },
+    "SelfServicePoliciesList": {
+      "description": "List of Self-service policies",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of policies",
+          "items": {
+            "$ref": "#/definitions/SelfServicePolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Pagination link"
+        }
+      }
+    },
+    "SystemData": {
+      "type": "object",
+      "description": "The system data",
+      "readOnly": true,
+      "properties": {
+        "createdBy": {
+          "type": "string",
+          "description": "Created by"
+        },
+        "createdAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Created at"
+        },
+        "lastModifiedBy": {
+          "type": "string",
+          "description": "last modified by"
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "type": "string",
+          "description": "last modified at"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "api-version": {
+      "in": "query",
+      "name": "api-version",
+      "description": "The api version to use.",
+      "required": true,
+      "type": "string",
+      "x-ms-client-name": "ApiVersion",
+      "x-ms-parameter-location": "method"
```

Comment:
Why `x-ms-parameter-location: method` for api-version? This typically goes on the client.

Reply:
Modified x-ms-parameter-location to client instead of method.